frontend/lib/Repository/EndpointRepository/RestEndpointService.dart

```diff
@@ -1,54 +1,55 @@
-import 'package:adpv_frontend/Repository/AbstractEndpointRepository.dart';
+import 'package:adpv_frontend/DataModels/EndpointSummary.dart';
 import 'package:dio/dio.dart';
-import '../Common/Consts.dart';
-import '../Common/URLs.dart';
-import '../DataModels/EnableField.dart';
-import '../DataModels/EndpointData.dart';
-import '../DataModels/EndpointSummary.dart';
 
-class BackendResponse<T>{
+import '../../Common/Consts.dart';
+import '../../Common/URLs.dart';
+import '../../DataModels/EnableField.dart';
+import '../../DataModels/EndpointData.dart';
+
+class BackendResponse<T> {
   final T data;
   final String error;
 
   BackendResponse(this.data, this.error);
 
-  BackendResponse.fromJson(Map json) :
-        data = json["data"],
+  BackendResponse.fromJson(Map json)
+      : data = json["data"],
         error = json["error"];
 }
 
-class RestEnpointRepository implements AbstractEndpointRepository {
-  final Dio client;
+class RestEndpointService {
```

Comment:
To bardziej pasowałoby na repozytorium restowe niż klasa `RestEndpointRepository`

Reply:
k